styleguide/drafts/tooling.rst

```diff
@@ -241,6 +242,30 @@ Alternatives
 
 * ¯\\_(ツ)_/¯ code formatters themselves (black, yapf, ..)
 
+Pyupgrade_
+++++++++++
+`Project Homepage <Isort_www_>`_
+
+Helps migrating old code (constructs) to modern one.
```

Comment:
General question, this is not only for python2 to python3, I guess. This is also between python3 versions

Reply:
No it is not only for `python2` to `python3`, it generally helps to update the code base to more modern constructs and api usages.

Example of `python 3.6` to `python 3.7`
```diff
-output = subprocess.run(['foo'], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+output = subprocess.run(['foo'], capture_output=True)

-output = subprocess.run(['foo'], universal_newlines=True)
+output = subprocess.run(['foo'], text=True)
```

Reply:
Nice